code/modules/events/grid_check.dm

```diff
@@ -9,7 +9,7 @@
 	startWhen = 1
 
 /datum/round_event/grid_check/announce(fake)
-	priority_announce("Abnormal activity detected in [station_name()]'s powernet. As a precautionary measure, the station's power will be shut off for an indeterminate duration.", "Critical Power Failure", 'sound/ai/poweroff.ogg')
+	priority_announce("Abnormal activity detected in [station_name()]'s power networks. As a precautionary measure, the sector's power will be remotely shut off for an indeterminate duration.", "Critical Power Failure", 'sound/ai/poweroff.ogg')
```

Comment:
```suggestion
	priority_announce("Abnormal activity detected in [station_name()]'s power networks. As a precautionary measure, the ship's power will be remotely shut off for an indeterminate duration.", "Critical Power Failure", 'sound/ai/poweroff.ogg')
```

Reply:
either way this should not happen outside of adminbus, so I'll flip a coin to add changes like this one. I call heads

Reply:
![image](https://user-images.githubusercontent.com/45698967/153486807-40e13f01-19af-4fa5-a1ac-b154270fd586.png)